.monorepolint.config.mjs

```diff
@@ -52,10 +52,28 @@ function standardPackageRules(shared) {
   return [
     standardTsconfig({
       ...shared,
+      excludePackages: [...shared.excludePackages, "@osdk/examples.basic.**"],
       options: {
         file: "tsconfig.json",
         template: {
-          extends: "tsconfig/base",
+          extends: "../../monorepo/tsconfig/tsconfig.base.json",
+
+          compilerOptions: {
+            rootDir: "src",
+            outDir: "build/types",
+            composite: true,
+          },
+          include: ["./src/**/*", ".eslintrc.cjs"],
+        },
+      },
+    }),
+    standardTsconfig({
```

Comment:
@ericanderson is there a way to make this less duplicitive?

Reply:
Either a function for reuse of the stnadardTsConfig. You can also give a generator function which passes you some details that might make it more programatic.

Reply:
Also this is so sad. The old includes were so much better!